app/views/courses/description.html.erb

```diff
@@ -42,7 +42,7 @@
 <% if @published.present? %>
   <div class="govuk-success-summary" aria-labelledby="success-summary-title" role="alert" tabindex="-1" data-module="error-summary">
     <h2 class="govuk-success-summary__title" id="success-summary-title">
-      Your course has been published
+      <%= @published %>
```

Comment:
I wonder if this should be `@success_message` as it is sometimes a save message and sometimes a publish one.

Reply:
I want to do another PR to refactor this to just use the flash